manager/controllers/app/select_modules.go

```diff
@@ -195,7 +195,7 @@ func (p *PlotterGenerator) findPathsWithinLimit(item *DataInfo, source *Node, si
 	for _, module := range p.Modules {
 		for capabilityInd, capability := range module.Spec.Capabilities {
 			// check if capability is allowed
-			if !allowCapability(item, capability.Capability) {
+			if !allowCapability(item, string(capability.Capability)) {
```

Comment:
Might be better to change allowCapability to accept taxonomy.Capability instead.

Reply:
changed